pkg/kubectl/cmd/cp/cp.go

```diff
@@ -338,7 +342,17 @@ func makeTar(srcPath, destPath string, writer io.Writer) error {
 }
 
 func recursiveTar(srcBase, srcFile, destBase, destFile string, tw *tar.Writer) error {
-	filepath := path.Join(srcBase, srcFile)
+	var reg *regexp.Regexp
+	var filepath string
+
+	isRegex := isRegexp(srcFile)
```

Comment:
Why you're using a regexp here where https://golang.org/pkg/path/filepath/#Glob exists?

Reply:
@soltysh Good point. Updated. PTAL. Thanks.